functions.lisp

```diff
@@ -39,6 +83,10 @@ arrays or objects."))
     (aref object index))
   (:method ((key string) (object hash-table))
     (gethash key object))
+  (:method ((pointer pathname) object)
+    (if (equal #p"" pointer)
+        object
```

Comment:
Same.

Reply:
Replied in the other comment. In short: `#p""`, as an empty pointer, returns the object itself.